pages/platform/kubernetes-k8s/etcd-quota-error/guide.de-de.md

```diff
@@ -154,6 +154,28 @@ kubectl get reportchangerequest.kyverno.io -A | wc -l
 If that still does not cover your case, you can use a tool like [ketall](https://github.com/corneliusweig/ketall) to easily list and count resources in your cluster.  
 Then you should delete the resources in excess and fix the process responsible for their creation.
 
+## Counting all resources
+
+If you still need to check all resources as you do not know what consumes etcd quotas, you can run this snippet. You will need the `count` plugin for `kubectl`:
```

Comment:
How can I install this plugin? ;-)
Through krew?
Please provide the way to install it 

thanks :)

Reply:
Yes you need to install it with krew. I'll update it.

Reply:
Added link to the installation instead of complete how-to install plugin+krew which is not revelant here.